docs/designers-developers/developers/tutorials/metabox/meta-block-3-add.md

```diff
@@ -53,9 +55,52 @@ Add this code to your JavaScript file (this tutorial will call the file `myguten
 		save: function() {
 			return null;
 		}
-	});
-})( window.wp );
+	} );
+} )( window.wp );
 ```
+{% ESNext %}
+```jsx
+import { registerBlockType } from '@wordpress/blocks';
+import { TextControl } from '@wordpress/components';
```

Comment:
I'm personally concerned about the change in the imports as it's something that doesn't work unless you define externals. While using globals signals that we should rely on `npm` packages to be bundled here. Think of any JavaScript developper reading this, I'm pretty sure they won't understand that this needs point to globals.

Reply:
That's a fair point I missed. Trying to put together my thoughts on this:

- **in tutorials, handbook, etc. we should promote globals**. These are tailored for WordPress developers, so the assumption is that they're building something to integrate with WordPress. Having both ES5 and ESNext is helpful for onboarding, but promoting globals makes sense here.

- **in package info (READMEs, JSDoc examples, etc) we should promote `import` syntax and avoid the globals**. Packages should be agnostic of the host they're bundled with (WordPress injecting the globals). It's not that important a goal to have good coverage for ES5 here.

This isn't perfect, though. For example: when we expose JSDoc examples in the handbook, they'll contain the `import` syntax. 

Does this sound like a good compromise? I can prepare a PR to fix this.

Reply:
For reference: things will be even more confusing in a few days, once we ship the zero-config `wp-scripts build` command that include externals (and the reason I thought about proposing this) :man_shrugging:

Reply:
> * **in package info (READMEs, JSDoc examples, etc) we should promote `import` syntax and avoid the globals**. Packages should be agnostic of the host they're bundled with (WordPress injecting the globals). It's not that important a goal to have good coverage for ES5 here.

Agreed on this one. I would be personally fine of using imports only for ESNext examples.

For tutorials and handbook in general, using `wp` globals is a very good idea.